# config_iot_ros.yaml            #"AKfycbzTctos0YodBJL99S2LPlhplDZClbcJfJIqOblZDVxyXHU4UPY"
# IoT Configuration
config_pyiot:
  mqtt:
    server_url: "broker.mqttdashboard.com"        # http://www.hivemq.com/demos/websocket-client/
    # server_url: "test.mosquitto.org"        # Alternative to HiveMQ
    server_port: 1883
    topic_sub: "eyrc/RpAaSgSa/iot_to_ros"          # <unique_id> = xYzqLm
    topic_pub: "eyrc/RpAaSgSa/ros_to_iot"          # <unique_id> = xYzqLm
    qos: 0

    sub_cb_ros_topic: "/ros_iot_bridge/mqtt/sub"   # ROS nodes can listen to this topic to receive data from MQTT
  google_apps:
    spread_sheet_id: "AKfycbzTctos0YodBJL99S2LPlhplDZClbcJfJIqOblZDVxyXHU4UPY"      #spreadsheet id1
    #spread_sheet_id2: "AKfycbw850dk4moVgebU2GGe0PUQUvvg8jTpSjBQCawJt3_13vgujLk"      #spreadsheet id2
